Workflow guide improvements (#31)
optakt/optakt.github.io: content/process/repositories.md

Once a PR has been reviewed and approved by at least one reviewer, it can be merged. The only way we allow merging is by squashing all the PR's commits into a single one, as this keeps the `master` branch and the changelog clean and easy to follow.

!!! warning "Closing Pull Requests"

>     When closing a Pull Request, make sure to always specify the reason for closing it by adding a comment to the PR while closing it.

Reviewer: ```suggestion
    When a PR needs to be closed, please make sure to always specify the reason for this action in a comment to the PR along with the closing.
```

Author: ```suggestion
    When closing a Pull Request, make sure to always specify the reason for doing so by adding a comment to the PR.
```

Reviewer: Yes, it works, but it is too conversational for a written text, especially in a guideline.
But anyway.

>     This will ensure that collaborators can follow the process and understand the reasons for the PR being closed, and ensures that the repository has a well-documented history.

Reviewer: ```suggestion
    This will help the collaborators follow the process and understand the reasons for closing the PR, as well as ensuring that the repository has a well-documented history.
```
Not sure if my suggestion is the best possible grammatically; the reason I made it is to avoid the repeating of "ensure" word in the same sentence.

Reviewer: Ok
